components/Features/FeatureVideoText.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import styles from './FeatureVideoText.module.scss';
+import FeatureVideo from './FeatureVideo';
+
+export default function FeatureaVideoText() {
+  return (
+    <div className={styles['video-text-section']}>
+      <div className={styles['video-text-wrapper']}>
+        <div className={styles['text-wrapper']}>
+          <h1 className={styles.heading}>Videos that engage</h1>
+          <p className={styles.paragraph}>
+            The most effective way to get people engaged and sharing online is
+            through a great story with a great video. This is why givngo focuses
+            on helping members create videos that tell their story in a way that
+            creates action. Our in house team of video creators and success
+            managers are always ready to givngo!
+          </p>
+        </div>
+        <div className={styles['video-wrapper']}>
+          <FeatureVideo
+            id="2F371994595"
```

Comment:
I don't think we are using this `id`. Maybe it is worth while to refactor `Video` so that we can use that in both places?